cmake/modules/autopas.cmake

```diff
@@ -17,7 +17,7 @@ if(ENABLE_AUTOPAS)
     ExternalProject_Add(
             autopas
             GIT_REPOSITORY ${autopasRepoPath}
-            GIT_TAG origin/master
+            GIT_TAG bayes-search-hyper
```

Comment:
undo this change

Reply:
Sure. This is only for Jan to work with this now.